hpx/runtime/components/server/runtime_support.hpp

```diff
@@ -160,10 +160,10 @@ namespace hpx { namespace components { namespace server
 
         /// \brief Actions to create new objects
         template <typename Component>
-        HPX_HOST_DEVICE naming::gid_type create_component();
+        naming::gid_type create_component();
 
```

Comment:
@hkaiser Possible but need to do some cleaner workaround (in terms of __host__ / __device__) around constructors/destructors of gid_type https://github.com/atrantan/hpx/blob/master/hpx/runtime/naming/name.hpp